eox_tenant/test/test_signals.py

```diff
@@ -283,3 +292,95 @@ def test_udpate_settings_with_existing_dict(self, _get_config_mock):
 
         self.assertEqual(settings.TEST_DICT_OVERRIDE_TEST.get("key1"), "Some Value")
         self.assertEqual(settings.TEST_DICT_OVERRIDE_TEST.get("key2"), "My value")
+
+
+class CelerySignalCLISyncTests(TestCase):
```

Comment:
@morenol  we are not testing the signals we are testing the receivers, so imo this should be called different